src/api/Feed.js

```diff
@@ -31,6 +31,14 @@ import {
     TASK_NEW_NOT_STARTED,
     TYPED_ID_FEED_PREFIX,
 } from '../constants';
+import type {
```

Comment:
It seems like these are our "model" types, is that correct? If so, should they instead live up here with the API flow types and be imported by the components? They could even be re-exported by the component types file, if desired.

Reply:
Hmm, I don't know. These types are used in multiple components and sometimes in the API files as well. Maybe so it doesn't become super complex with types all over the place and re-exporting we should go back to plan B and have something like this, with all the types related to tasks in one file independent of components and API? This approach could be used for types that span multiple components and/or API:

```
src/common/types
  core.js (what used to be src/common/box-types.js)
  tasks.js
  ... other files in the future that have types spanning multiple separate things
```

Reply:
Sounds like a good compromise.

Reply:
Done.